doc-examples/r2dbc-multitenancy-schema-example-java/build.gradle

```diff
@@ -20,19 +23,16 @@ dependencies {
     annotationProcessor projects.dataDocumentProcessor
 
     implementation mnMultitenancy.micronaut.multitenancy
-    implementation 'io.micronaut.reactor:micronaut-reactor'
-    implementation 'io.micronaut:micronaut-http-client'
-    implementation 'io.micronaut.data:micronaut-data-r2dbc'
+    implementation mnReactor.micronaut.reactor
```

Comment:
The test in the doc-examples was fixed only when I changed build.gradle and the way dependencies were imported.